frontend/elements/src/title-sub-paragraph.ts

```diff
@@ -0,0 +1,81 @@
+import { MEDIA_UI } from '@utils/path';
```

Comment:
we have a `title` area in `core` - please move this there.

Reply:
This has been moved